clevercsv/normal_form.py

```diff
@@ -62,7 +62,7 @@ def detect_dialect_normal(
             return None
 
     form_and_dialect: List[
-        Tuple[int, Callable[[str, SimpleDialect], bool], SimpleDialect]
+        Tuple[int, Callable[[list[str], SimpleDialect], bool], SimpleDialect]
```

Comment:
I think the build is failing because you need `from typing import List` here for Python 3.8.

Reply:
Oh, that is probably it, I am so used to the liwercase versions I did not think of it :) thank you for the patience, I will fix it as soon as I can